### PR TITLE
Cleanup indexer

### PR DESCRIPTION
**Pull request by shaedrich**

### CODE REVIEW

File: utilities/indexer.cfm [hunk 1]
```diff
@@ -105,10 +93,21 @@
...
 	<cfset arguments.json = replace(arguments.json, "}", "#chr(10)#}", "all")>
+	
 	<cfreturn arguments.json>
 </cffunction>
+
+<cffunction name="writeRelated" returnType="boolean" output="false">
+	<cfargument name="file" type="string">
+	
+	<cfset var JSON = fileRead(dataDir & "/" & file & ".json", "utf-8")>
+	<cfset var var data = deserializeJSON(JSON)>
```
Comment (pfreitag) [inline utilities/indexer.cfm]: two `var` statements here

Comment (shaedrich) [reply utilities/indexer.cfm]: Done.

**Comment by pfreitag** [comment]
closing this PR - don't like how the writeRelated function is working with the outer variables scope here.

**Comment by shaedrich** [comment]
@pfreitag I could pass them to form scope

**Pull request closed without merging**